game_template/lib/src/audio/songs.dart

```diff
@@ -0,0 +1,20 @@
+const Set<Song> songs = {
+  // Filenames with whitespace break package:audioplayers on iOS
+  // (as of February 2022), so we use no whitespace.
+  Song('Mr_Smith-Azul.mp3', 'Azul'),
+  Song('Mr_Smith-Sonorus.mp3', 'Sonorus'),
+  Song('Mr_Smith-Sunday_Solitude.mp3', 'SundaySolitude'),
+};
+
+class Song {
+  final String filename;
+
+  final String author = 'Mr Smith';
+
+  final String name;
+
+  const Song(this.filename, this.name);
+
+  @override
+  String toString() => 'Song<$filename>';
+}
```

Comment:
Maybe adjust `author` to `artist` with an optional initializing parameter to make the class slightly more flexible/generic.

```suggestion
const Set<Song> songs = {
  // Filenames with whitespace break package:audioplayers on iOS
  // (as of February 2022), so we use no whitespace.
  Song('Mr_Smith-Azul.mp3', 'Azul', artist: 'Mr Smith'),
  Song('Mr_Smith-Sonorus.mp3', 'Sonorus', artist: 'Mr Smith'),
  Song('Mr_Smith-Sunday_Solitude.mp3', 'SundaySolitude', artist: 'Mr Smith'),
};

class Song {
  final String filename;

  final String name;

  final String? artist;

  const Song(this.filename, this.name, {this.artist});

  @override
  String toString() => 'Song<$filename>';
}
```